### ISSUE TITLE
Issue when rendering server side with react (checksum warning)

### ISSUE THREAD

**Issue by swernerx** [ReactTraining/react-router#4501]
It seems like `NavLink` is behaving differently on client and server.

I have this code in my application:

```js
import { Switch, Route, NavLink } from "react-router-dom"

...

<ul>
  <li><NavLink to="/" activeClassName={Styles.activeLink}>Home</NavLink></li>
  <li><NavLink to="/about" activeClassName={Styles.activeLink}>About</NavLink></li>
</ul>
```

In the markup this seems to produce double slashes for `href`:

```html
<ul data-reactid="7">
<li data-reactid="8"><a class="src-app-AppContainer-activeLink " href="//" data-reactid="9">Home</a></li>
<li data-reactid="10"><a href="//about" data-reactid="11">About</a></li>
</ul>```

**Comment by pshrmn** [comment]
Fixed in #4484

**Comment by swernerx** [comment]
Ah perfect 👍
